### ISSUE TITLE
Ha-bridge web interface on pi not working

### ISSUE THREAD

**Issue by rcpett2** [bwssytems/ha-bridge#328]
Just got a raspberry pi 3 and installed 3.5.1.  When I start up the web interface 192.168.1.26:8080, the only that comes up is the top banner "Home My Echo Help About."  The rest of the screen is blank.  No Bridge Devices, Bridge Control, or anything else.  I have tried using the bridge on the hardwired and on the wireless interfaces. I have jdk 8. (Network works fine on either interface.) 

Any help would be much appreciated.  This is so frustrating... 

Thanks
Rob

P.S.  3.5.1 jar works fine on my macbook pro.

**Comment by relighted** [comment]
I'm running on a stock raspberian /pixel cars image as well,  this has Java8 on board by default.   I had to run the jar with sudo, and it's default is on port 80. If you run on 8080 it does not look standard. 

Running it as service did bring a challenge as the data folder wasn't used and writable.

**Comment by bwssytems** [comment]
How are you starting the ha-bridge? Service or command line or script?

**Comment by rcpett2** [comment]
Tried starting it both service and command line, same result on the screen.

Here is what I found since I posted the original comment:  I tried accessing the web interface on the pi from a different computer (my macbook pro) on which I use Chrome as my browser.  It comes up fine and everything is working properly - still blank on the pi browser.  I assume that it is the default browser on the pi that is somehow not compatible with the ha-bridge web interface.  Does that sound familiar?
